src/main/java/org/aksw/simba/squirrel/sink/TripleBasedSink.java

```diff
@@ -3,14 +3,18 @@
 import org.aksw.simba.squirrel.data.uri.CrawleableUri;
 import org.apache.jena.graph.Triple;
 
+import java.util.List;
+
 /**
  * A sink that can handle triples.
- * 
+ *
  * @author Michael R&ouml;der (michael.roeder@uni-paderborn.de)
  *
  */
 public interface TripleBasedSink extends SinkBase {
 
     public void addTriple(CrawleableUri uri, Triple triple);
 
+    List<Triple> getTriplesForGraph(CrawleableUri uri);
```

Comment:
This is not that good. The problem is that not all sinks will be able to easily access the data that they have written :disappointed: